src/components/team/TeamMemberSelectionContext.tsx

```
@@ -0,0 +1,732 @@
+      return { ...state, searchQuery: action.payload };
+    case 'SET_NICKNAME':
+      return { ...state, nickname: action.payload };
+    case 'SET_PREVIEW_NICKNAME':
+      return { ...state, previewNickname: action.payload };
+    case 'RESET_STATE':
+      return { ...initialState, activeSlot: 'head' };
+    case 'SET_AVAILABLE_POKEMON':
+      return { ...state, availablePokemon: action.payload };
+    case 'INITIALIZE_FROM_EXISTING':
+      const { headPokemon, bodyPokemon, headLocationId, bodyLocationId } =
+        action.payload;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)